README.md

```diff
@@ -1,31 +1,59 @@
 
-**To be added: We also need some explanation of the high-level module structure of the code. The information to be conveyed is this
+This project was made using TypeScript as the main programming language, React Native for the frontend technology, Firebase for our backend technology, and Expo for our development environment. Our final product combines the two apps created by the previous team for fans and admins.
+
+You can find information about the structure and design of our app in our [Design Document](https://tarheels.live/comp523teami/architecture-diagram/)!
+
+## Accessing and Running FanzPlay
+
+For detailed information on how to run the app, please refer to both the User Manual and the Adminstrator Manual on our app website in the section of [Documentation Plan](https://tarheels.live/comp523teami/documentation-plan/).
+
+You can also refer to your [Handoff Plan](https://tarheels.live/comp523teami/handoff-plan/) which includes helpful videos on how to access and run the app.
+
+## Implementation details
+
+For an overview of the implementation, code interactions, and discussed limitations please view the docs folder:
+
+- [Admin implementation supplementary documentation](https://github.com/stefanodongowski/FanzPlay/blob/stage/docs/admin.md)
+- [User implementation supplementary documentation](https://github.com/stefanodongowski/FanzPlay/blob/stage/docs/user.md)
 
-what are the programming languages used in your code?
-in relation to the architecture diagram, what parts are written in which languages?
-What are the major modules in each component? This may well be satisified by the information generated by the repo. What are the relationships among the modules? You may find an annotated version of your architecture diagram a useful way to convey this information.**
+## Testing
 
-You can find more information about the structure and design of our app in our [design document]()!
+Check out our [Test Plan](https://tarheels.live/comp523teami/test-plan/)! This includes tests we planned and tests we were able to do. You can use this as a guide to the testing requirements of the app in general.
 
-## Running the App
-To run the app, run the command `npx expo start` in the terminal. 
+The testing is done inside the [test folder](https://github.com/stefanodongowski/FanzPlay/tree/stage/test) in this repo.
 
 ## Linting
-Before merging your branch, be sure to lint the code by running 
-`npm run lint` and `npm run prettier:write` in the terminal. 
 
+Before merging your branch, be sure to lint the code by running
+`npm run lint` and `npm run prettier:write` in the terminal. This will properly format your code and lint to maintain code readability and consistency.
+
+## Maintabality
```

Comment:
I think maintainability may be spelled wrong?

Reply:
You are correct. Fixed now